demo/sm_42v_1.tscn

```diff
@@ -31,6 +31,20 @@ oil_pump/pressure_minimum = 0.15
 [node name="TestowySilnikElektryczny" type="TrainElectricSeriesEngine" parent="."]
 nominal_voltage = 11000.0
 winding_resistance = 200.0
+power/source = 5
```

Comment:
ponieważ stonka (demo/sm_42_v1.tscn) nie ma elektrycznego silnika a już teraz mapowane są parametry jakiegoś prawdziwego loka, to rozważyłbym wybranie sobie FIZ-a elektryka i zrobienie nowego tscn. 

W głównym demku dodamy switcher loków.